app/javascript/src/javascripts/uploader/preview_image.vue

```diff
@@ -0,0 +1,13 @@
+<template>
```

Comment:
combine these two templates into a single component. Accept an object prop with the shape `{isVideo: boolean, url: string}`

Reply:
Done, but I just moved both into the existing component. There wasn't much use for it to be separate anyways.